internals/secrethub/migrate.go

```diff
@@ -450,38 +434,79 @@ func (cmd *MigrateApplyCommand) Run() error {
 				return err
 			}
 		} else {
-			if !cmd.append {
-				confirmed, err := ui.AskYesNo(cmd.io, fmt.Sprintf("A vault named %s already exists. Do you want to append the items to that vault?", vault.Name), ui.DefaultYes)
-				if err != nil {
-					return fmt.Errorf("a vault named %s already exists. Use --apend to append to existing vaults", vault.Name)
-				}
-				if !confirmed {
-					fmt.Fprintln(cmd.io.Output(), "Aborting.")
-					return nil
-				}
-			}
-
-			fmt.Fprintf(cmd.io.Output(), "[%d/%d] Appending to existing vault: %s\n", i, len(plan.vaults), vault.Name)
+			fmt.Fprintf(cmd.io.Output(), "[%d/%d] Updating vault: %s\n", i, len(plan.vaults), vault.Name)
```

Comment:
Yes, the message indeed doesn't make sense. I wanted to output something to indicate progress, as the migration tool can be slow and it would be nice to have output even if we don't need to update any values in a vault. I'd say we should prompt something like `Checking vault: <vault-name>` or maybe just the vault name to indicate progress. I'm afraid that if we only output when we actually change something the user gets no indication of progress if all secrets are up to date. What do you think?

Reply:
Yes, that sounds reasonable. Probably makes even more sense now that we plan on splitting the checking and actual applying?